### ISSUE TITLE
method 

### ISSUE THREAD

**Issue by yunzqq** [JasonSWFu/MetricGAN#5]
it is operate on TF domain right?

**Comment by JasonSWFu** [comment]
Yes

**Comment by yunzqq** [comment]
> Yes

Thks

**Comment by yunzqq** [comment]
For doing comparison in the voice bank + DEMAND dataset, i want to know the parameter size of Metric GAN (G or D each rescpectively)   thks  very much!

**Comment by yunzqq** [comment]
Dear Szu-Wei Fu

I am a student from harbin institute of technology.
I am doing research on speech enhancement.

Recently, i am reading your paper Metric GAN speech enhancement.

I want to consult some issues (1) how many parameters for G and D networks, respectively? (2) G network use BLSTM netowork, so it is a non-causal system, right?

Thank you very much for your help!
I am looking forward to your reply.

Best,
Qiquan Zhang, 
HIT


949245661@qq.com
 
From: Szu-Wei Fu
Date: 2019-11-07 15:17
To: JasonSWFu/MetricGAN
CC: George_Zhang; Author
Subject: Re: [JasonSWFu/MetricGAN] method (#5)
Yes

**Comment by JasonSWFu** [comment]
Hi Qiquan,

1) After checking model.summary(), the number of parameters for G and D are 1,892,057
and 690,855, respectively.
2) Yes, it is non-causal.

Sincerely, 
Szu-Wei

**Comment by yunzqq** [comment]
> Hi Qiquan,
> 
> 1. After checking model.summary(), the number of parameters for G and D are 1,892,057
>    and 690,855, respectively.
> 2. Yes, it is non-causal.
> 
> Sincerely,
> Szu-Wei


Hi, Szu-Wei
Thanks, it is same as i check,   thank you !

Sincerely,
Qiquan

**Comment by yunzqq** [comment]
and part of parameters in D network are not learnable.

**Comment by yunzqq** [comment]
Dear Szu-Wei Fu,

We are making comparison with MetricGAN on the DEMAND database.
For results of metrics, we use the value reported in the paper.
But, we want to do a comparison on spectrograms of speech.
So, could you supply the enhanced speech or pre-trained model for us?

Thank you very much for your help！

Best,
Qiquan Zhang
Harbin Institute of Technology


949245661@qq.com
 
From: Szu-Wei Fu
Date: 2019-12-02 14:14
To: JasonSWFu/MetricGAN
CC: George_Zhang; Author
Subject: Re: [JasonSWFu/MetricGAN] method (#5)
Hi Qiquan,
After checking model.summary(), the number of parameters for G and D are 1,892,057
and 690,855, respectively.
Yes, it is non-causal.
Sincerely,
Szu-Wei
